### ISSUE TITLE
Allow installation of Aliucord on other Android user profiles

### ISSUE THREAD

**Issue by Mr-Fred-Flanders** [Aliucord/plugin-requests#1152]
### Discord Account

FredFlanders

### Plugin Description

To allow installation of Aliucord on other Android user profiles.

### Go into more detail...

I was not able to find a section on this Github to make a feature request, so this seemed like the next best option.

After a lengthy chat with a very patient and helpful support member, I discovered that the Aliucord Installer is unable to finish the process when attempting to install on a profile that isn't the "Owner" profile. While for most users, this may not be an issue. _But for GrapheneOS users_, this is unfortunate, as using a different profile is a common security practice and having to switch to the Owner profile just to use Discord (Aliucord) is not ideal.

TLDR: **If possible, please consider allowing the installation on other profiles! It would be really appreciated.**

### Request Agreement

- [X] I have made sure my plugin is possible and abides by the rules!

### ​

- [X] I did indeed check to make sure my plugin request is original!

**Comment by OmegaSunkey** [comment]
not possible with a plugin as this is before installing aliucord

**Comment by rushiiMachine** [comment]
ref: https://github.com/Aliucord/Aliucord/issues/261
